### ISSUE TITLE
[BUG] Navigating Albums

### ISSUE THREAD

**Issue by Aamir3d** [RupertAvery/DiffusionToolkit#225]
Technically not a bug, but there is no way to navigate (Go UP) or select Albums like folders right now. There should be a Navigate Up or "Home" button for albums to get to the top.

Example: I created some albums, and went into one of them to browse images.
Now, there is no way to get to another album without exiting the application.
![image](https://github.com/RupertAvery/DiffusionToolkit/assets/122131887/9881a811-52a4-41ca-b8da-de5d69c65fc2)

**Comment by RupertAvery** [comment]
You're right, though. the navigation pane (view > albums) is supposed to make this easier.

**Comment by Aamir3d** [comment]
> You're right, though. the navigation pane (view > albums) is supposed to make this easier.

Ah I just checked that. Might want to make it default behavior when the albums button is ticked? Or is there a difference?
